aries_cloudagent/indy/credx/holder.py

```diff
@@ -379,8 +379,15 @@ async def credential_revoked(
         """Check ledger for revocation status of credential by cred id.
 
         Args:
-            credential_id: Credential id to check
+            ledger (BaseLedger): The ledger to check for revocation status.
+            credential_id (str): The ID of the credential to check.
+            fro (int, optional): The starting sequence number of the revocation registry
```

Comment:
Do we care this should be “from”?

Reply:
Ya. I'll change this here. It was like this before, but no point in truncating a word this short.

Reply:
So, this was actually done on purpose because `from` is a reserved keyword in python. I'm going to change these arguments to `timestamp_from` and `timestamp_to` instead.